### ISSUE TITLE
demo.titon.io - Modal - Form Posting - Error Line cut in 2 horizontally

### ISSUE THREAD

**Issue by TraderStf** [titon/toolkit#113]
FYI

In [Modal](http://demo.titon.io/?modal&rtl=0), click Form Posting, click Submit and on top of the page the message
"Something terrible has happened!"
appears in a rectangle, white on blue background, with a closing cross on far right.

But on Mac OS X - Safari / FireFox / Chrome / Opera. All latest version, no gadgets or plugin.
<i>If you want me to test other browsers, ping me :+1: </i>

The rectangle has the height of the text + 4~ pixels above and below and the cross has its center point vertically on the bottom of the rectangle, horizontally it is ok, on the far right.

![modalformposting](https://cloud.githubusercontent.com/assets/5954335/6827866/9657cfec-d30a-11e4-86d2-3a6e4a74129a.png)

In other words, that rectangle is half the height of those visible in the "Animations. Clickout disabled.".

**Comment by milesj** [comment]
Ah thanks for pointing this out, it seems like the AJAX call is failing.
